explore/future_compat.h

```diff
@@ -54,20 +54,23 @@
 // Add new ignored warnings here:
 #if defined(_MSC_VER)
     #define VW_WARNING_DISABLE_DEPRECATED_USAGE    VW_WARNING_DISABLE(4996)
-    #define VW_WARNING_DISABLE_CLASS_MEMACCESS
-    #define VW_WARNING_DISABLE_CAST_FUNC_TYPE
+#  define VW_WARNING_DISABLE_BADLY_FORMED_XML VW_WARNING_DISABLE(4635)
+#  define VW_WARNING_DISABLE_CLASS_MEMACCESS
+#  define VW_WARNING_DISABLE_CAST_FUNC_TYPE
 #elif defined(__GNUC__) || defined(__clang__)
     #define VW_WARNING_DISABLE_DEPRECATED_USAGE   VW_WARNING_DISABLE(-Wdeprecated-declarations)
+#  define VW_WARNING_DISABLE_BADLY_FORMED_XML
 
     // This warning was added in GCC 8
-    #if __GNUC__ >= 8
-        #define VW_WARNING_DISABLE_CLASS_MEMACCESS    VW_WARNING_DISABLE(-Wclass-memaccess)
-        #define VW_WARNING_DISABLE_CAST_FUNC_TYPE    VW_WARNING_DISABLE(-Wcast-function-type)
-    #else
-        #define VW_WARNING_DISABLE_CLASS_MEMACCESS
-        #define VW_WARNING_DISABLE_CAST_FUNC_TYPE
-    #endif
+#  if __GNUC__ >= 8
+#    define VW_WARNING_DISABLE_CLASS_MEMACCESS VW_WARNING_DISABLE(-Wclass - memaccess)
```

Comment:
Does this still work?

Reply:
I believe it does, preprocessor should remove whitespace